doc/website/blog/2023-07-03-versioned-migrations-kubernetes.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Versioned migrations support for the Atlas Kubernetes Operator"
+authors: rotemtam
+tags: [migrations, kubernetes, operators]
+---
+:::info TL;DR
+
+You can now use the Atlas Kubernetes Operator to run versioned migrations natively from
+your Kubernetes cluster.
+
+[See an example](#demo-time)
+
+:::
+
+## Introduction
+
+The Atlas Kubernetes Operator is a Kubernetes operator that enables you to manage
+your database schemas natively from your Kubernetes cluster.  By exposing custom
+resource definitions (CRD) the operator extends the Kubernetes API to support
+database schema management.
+
+In a [previous blog post](/2023/05/08/atlas-v011-kubernetes-operator) we demonstrated
+how to use the Atlas Operator for the [declarative (state-based) workflow](/integrations/kubernetes/operator#declarative-schema-migrations)
+in which you define the desired state of your database schema in a Kubernetes manifest
+and the operator takes care of the rest.
+
+State vs versioned based migrations is a common and unresolved debate in the database schema
+management world, and we built Atlas to [support both](/concepts/declarative-vs-versioned) from
+the get-go.  
+
+Today, we are happy to announce [v0.2.0](https://github.com/ariga/atlas-operator/releases/tag/v0.2.0) 
+of the Atlas Kubernetes Operator which adds support support for the 
+[versioned migration workflow](/integrations/kubernetes/operator#versioned-schema-migrations).
+
+In this blog post we will demonstrate how to use the Atlas Operator for the 
+with this new workflow.
+
+## How it works
+
+![](https://atlasgo.io/uploads/k8s/operator/versioned-flow.png)
+
+The Atlas Kubernetes Operator supports [versioned migrations](https://atlasgo.io/concepts/declarative-vs-versioned#versioned-migrations).
+In versioned migrations, the database schema is defined by a series of SQL scripts ("migrations") that are applied
+in lexicographical order. The user can specify the version and migration directory to run, which can be located
```

Comment:
```suggestion
in order. The user can specify the version and migration directory to run, which can be located
```
Most tools (we as well) follow lexicographical ordering, but Flyway and Liquibase allow to do it differently.